user/docs/server.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Installing a NeoForge Server
+
+_This article assumes that you [have the correct version of Java installed][java]._
+
+## Installing
```

Comment:
Are we assuming that no one will do this on macos?

Reply:
My thought process is someone doing those old youtube tutorials that tells you how to get a server setup on your local machine to which you port forward and share your ip with your friends.